Remove "set but not used" variable reported by gcc-4.6 (fails with -Werror)

--- include/jrl/mal/boostmatrix.hh~	2011-11-07 12:02:59.000000000 +0100
+++ include/jrl/mal/boostmatrix.hh	2011-11-07 12:04:49.467456209 +0100
@@ -301,7 +301,6 @@
 	    ::boost::numeric::ublas::vector<FloatType> s(std::min(NR,NC));
 	    char Jobu='A'; /* Compute complete U Matrix */
 	    char Jobvt='A'; /* Compute complete VT Matrix */
-	    char Lw; Lw='O'; /* Compute the optimal size for the working vector */
 
 
 	    {
@@ -426,7 +425,6 @@
 	    ::boost::numeric::ublas::vector<FloatType> s(std::min(NR,NC));
 	    char Jobu='A'; /* Compute complete U Matrix */
 	    char Jobvt='A'; /* Compute complete VT Matrix */
-	    char Lw; Lw='O'; /* Compute the optimal size for the working vector */
 
 	    /* Get workspace size for svd. */
 	    {
